src/_includes/layouts/hardware.njk

```diff
@@ -1,44 +1,74 @@
 ---
 layout: layouts/documentation.njk
 ---
+{% set ramValue = specifications.RAM or '0' %}
+{% set ramString = ramValue | string %}
+
+{% if 'GB' in ramString %}
```

Comment:
Or the `specifications.RAM` is moved to `specifications.ram_mb` and it's always an integer